infra/gcp/lib.sh

```diff
@@ -275,6 +275,11 @@ function empower_group_for_gcb() {
         projects add-iam-policy-binding "${project}" \
         --member "group:${group}" \
         --role roles/cloudbuild.builds.editor
+
+    gcloud \
```

Comment:
Can you add a TODO: comment here saying this is just for `serviceusage.services.use` permission?

At this point we should refactor it to create a role with the perms we need and just bind to that.

Reply:
@thockin -- Added!